docs/guides/high-availability/operate-failover.adoc

```diff
@@ -0,0 +1,32 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Fail over to the secondary site"
+summary="This describes the automatic and operational procedures necessary"
+preview="true" >
+
+This {section} describes the steps to fail over from primary site to secondary site in a setup as outlined in <@links.ha id="concepts-active-passive-sync" /> together with the blueprints outlined in <@links.ha id="bblocks-active-passive-sync" />.
+
+== When to use procedure
+
+A failover from the primary site to the secondary site will happen automatically based on the checks configured in the loadbalancer.
+
+When the primary site loses its state in {ispn} or a network partition occurs that prevents the synchronization, manual procedures are necessary to recover the primary site before it can handle traffic again, see the <@links.ha id="operate-switch-back" /> {section}.
```

Comment:
{ispn} shown in rendered output:
![image](https://github.com/keycloak/keycloak/assets/2271511/d606b905-1af7-4a68-99e8-da603b67458a)

Reply:
fixed in next commit